tutorial_submission_policy.md

```diff
@@ -0,0 +1,64 @@
+# PyTorch Tutorial Submission Policy
+
+This policy outlines the criteria and process for submitting new tutorials to the PyTorch community.
+Our goal is to ensure that all tutorials are of high quality, relevant, and up-to-date, supporting
+both the growth of our users and the evolution of the PyTorch framework itself. By following these
+guidelines, contributors can help us maintain a robust and informative educational environment.
+
+## Acceptance Criteria For New Tutorials
+We are committed to enhancing the learning experience for PyTorch users by providing high-quality
+tutorials. We accept new tutorials that adhere to one of the following use cases:
+
+* **Support for New PyTorch Features:** Tutorials that support new features for upcoming PyTorch
+releases are typically authored by the engineers who are developing these features. These tutorials
+are crucial for showcasing the latest advancements in PyTorch.
+
+* **Tutorials from PyTorch Partners:** We welcome tutorials from our partners who are actively
+  collaborating with us to grow the PyTorch ecosystem.
+  
+* **Direct Requests from PyTorch Maintainers:** If a PyTorch maintainer has directly reached out
+  to you to request a tutorial or otherwise, endorsed a tutorial, this indicates a specific need
+  within the community or the project.
+  
+* **Tutorials from the open-source community members:** We accept tutorials from the community
+  members. However, as we can't accept all of them, we have identified the process below.
+  Please read below before sending a PR.
+
+The first three use cases do not require following the submission process described below.
+If your tutorial falls under the fourth category, please read and follow the instructions in
+the **Submission Process For Community-Contributed Tutorials** section.
+
+## Submission Process For Community-Contributed Tutorials
+
+To maintain the quality and relevance of tutorials, community-contributed tutorials must undergo
+a review process. If you are interested in contributing a tutorial, please follow these steps:
+
+1. **Create an issue:**
+   * Open an issue in the pytorch/tutorials repository proposing the new tutorial. Clearly
```

Comment:
Do you have an issue template with directed questions they should be answering in their proposal? That might help get consistent submissions.

Reply:
I do have a template for requesting a new tutorial: https://github.com/pytorch/tutorials/blob/main/.github/ISSUE_TEMPLATE/feature-request.yml I can repurpose it a bit to make it both request and proposal